lib/Importer/ONNX.cpp

```diff
@@ -80,6 +80,21 @@ size_t getPad(ArgumentDictionaryTy &dict) {
   return 0;
 }
 
+std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
```

Comment:
can you also avoid duplication with the getPad method?

Reply:
getPad() is for padding in pool operators now, and later it will be removed after supporting non-square padding for the pool ops. So I think it is OK to leave the duplication there. Otherwise, after getPad() is removed, getPads() needs to be updated again I think.

Reply:
As long as the format for pooling ops is the same, getPad can call getPads and return [0] element.